### PR TITLE
[BIOMAGE-1209] - Fix Matomo Heatmap

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-1209

#### Link to staging deployment URL 

https://ui-agi-ui425.scp-staging.biomage.net/

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [x] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/utils/tracking.js [hunk 1]
```diff
@@ -14,11 +14,11 @@ const trackingInfo = {
...
   },
   [Env.STAGING]: {
-    enabled: false,
+    enabled: true,
     siteId: 2,
   },
   [Env.DEVELOPMENT]: {
-    enabled: false,
+    enabled: true,
     siteId: 3,
```
Comment (aerlaut) [inline src/utils/tracking.js]: I'll revert the 2 enabled after a review of the staging environment

File: src/utils/tracking.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { init, push } from '@socialgouv/matomo-next';
 import { Auth } from 'aws-amplify';
 import Env from './environment';
 
-const MATOMO_URL = 'https://biomage.matomo.cloud/';
+const MATOMO_URL = 'https://biomage.matomo.cloud';
```
Comment (aerlaut) [inline src/utils/tracking.js:5]: The issue with heatmap not showing up is here. The problem is, the `socialguv/matomo-next` plugin that we're using adds an extra `/` for paths. See https://github.com/SocialGouv/matomo-next/blob/master/src/index.ts#L77

**Comment by aerlaut** [comment]
To see the heatmaps, you can go to [Matomo](biomage.matomo.cloud), choose the "STAGING DEVELOPMENT" environment and then look at the heatmap.

**Review by kafkasl — APPROVED** [review]

File: src/utils/tracking.js (hunk 1 above)
Comment (aerlaut) [reply src/utils/tracking.js]: reverted

**Pull request merged**